extra-network/kubo/spec

```diff
@@ -0,0 +1,3 @@
+VER=0.18.1
+SRCS="tbl::https://github.com/ipfs/kubo/archive/refs/tags/v$VER.zip"
+CHKSUMS="sha256::4dc65c1177fe2ff339c64063643276112efa029117d0810f14422208dd6a3c32"
```

Comment:
You should add a CHKUPDATE field indicating a way to automatically check for updates. This is done with [aosc-findupdate](https://github.com/AOSC-Dev/aosc-findupdate). We usually use the `anitya` type, using Fedora's [Anitya](https://release-monitoring.org/).

Please register an account there and submit `kubo` as one of the software to track. After you are done, you will get an update-check ID. For instance, Bash, at https://release-monitoring.org/project/166/, means that it has an ID of 166.

In this case, then, the CHKUPDATE line should go something like this.

```
CHKUPDATE="anitya::id=166"
```

Reply:
We require this information for all packages, where possible.